web-components/src/components/fixed-footer/BuyFooter.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import { makeStyles, useTheme, DefaultTheme as Theme } from '@mui/styles';
+import { Box, useTheme } from '@mui/material';
```

Comment:
`useTheme` should be imported from `/material` (not `/styles`) to get proper typing